src/tests/system/tests/test_passkey.py

```diff
@@ -411,3 +411,83 @@ def test_passkey__su_same_key_for_multi_user(
             ioctl=f"{moduledatadir}/umockdev.ioctl",
             script=f"{testdatadir}/umockdev.script.{suffix}.{user}",
         )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.ticket(jira="SSSD-7011", gh=7066)
+@pytest.mark.topology(KnownTopologyGroup.AnyAD)
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.builtwith(client="passkey", provider="passkey")
+def test_passkey__ssh_key_as_passkey_mapping_token(
+    client: Client, provider: GenericADProvider, moduledatadir: str, testdatadir: str
+):
+    """
+    :title: Check ssh key as a passkey mapping token with AD, Samba and LDAP server.
+    :setup:
+        1. Add a users in AD, Samba and LDAP server and add ssh key as a passkey mapping.
+        2. Setup SSSD client with FIDO, start SSSD service.
+    :steps:
+        1. Check su non-passkey authentication of the user.
+        2. Required error message in pam log.
+    :expectedresults:
+        1. Successfully su authenticate the user with correct password.
```

Comment:
This phrase and the code have me perplexed. I don't think the phrase explains correctly what you are doing.

Reply:
could be okay like this